@xen-orchestra/web/src/pages/vm/[id]/index.vue

```diff
@@ -0,0 +1,6 @@
+<script lang="ts" setup>
+definePage({
+  name: '/host/:id',
+  redirect: (to: any) => `/host/${to.params.id}/dashboard`,
+})
```

Comment:
Be careful to use the right route:
```suggestion
definePage({
  name: '/vm/:id',
  redirect: (to: any) => `/vm/${to.params.id}/console`,
})
```
As a side effect, this will fix the change in `typed-router.d.ts`